hal/network/ncp/wifi/wlan_hal.cpp

```diff
@@ -288,15 +289,30 @@ void SPARK_WLAN_SmartConfigProcess() {
 void HAL_WLAN_notify_simple_config_done() {
 }
 
+static WLanSelectAntenna_TypeDef current_antenna = ANT_AUTO;
+
 int wlan_select_antenna(WLanSelectAntenna_TypeDef antenna) {
-    if (antenna != ANT_AUTO) {
-        return SYSTEM_ERROR_NOT_SUPPORTED;
+    radio_antenna_type new_antenna = RADIO_ANT_UNKNOWN;
```

Comment:
Is there a problem adding this? or should I avoid declaring a new `RADIO_ANT_UNKNOWN` enum value?

Reply:
Doesn't look like it's going to be used anywhere else as of right now, might be better to just error out in `else` below.